lib/snapshot.js

```diff
@@ -24,6 +24,7 @@ const GENERAL_ERROR       = 6;
 
 const EXIT_UNRESPONSIVE   = 4;
 const EXIT_ERROR_LIMIT    = 5;
+const DEFAULT_FORMAT 	  = 'jpeg'
```

Comment:
No `;`. Although it will work, let's keep it clean. :)

Reply:
Also, what's the reason this needs to be `jpeg` instead of just `jpg`? It seems like it would simplify the code in some places where we need to specify the file extension for this particular format.

Reply:
I'm cool as long as we don't change the file extension, since it will invalidate the whole cache which is a problem.

Reply:
Exactly. Puppeteer will only accept jpeg.